cmd/pipeline/deploy.go

```diff
@@ -142,61 +146,70 @@ func (pc *Command) ExecuteDeployPipeline(ctx context.Context, opts *DeployOption
 		return fmt.Errorf("failed to load okteto context '%s': %v", okteto.Context().Name, err)
 	}
 
-	if opts.SkipIfExists {
-		cfgName := pipeline.TranslatePipelineName(opts.Name)
-
-		cfg, err := configmaps.Get(ctx, cfgName, opts.Namespace, c)
-		if err != nil && !oktetoErrors.IsNotFound(err) {
+	exists := false
+	cfgName := pipeline.TranslatePipelineName(opts.Name)
+	cfg, err := configmaps.Get(ctx, cfgName, opts.Namespace, c)
+	if err != nil {
+		if opts.ReuseParams && oktetoErrors.IsNotFound(err) {
+			return fmt.Errorf("Dev Environment not found to reuse params. Deploy your pipeline without --reuse-params")
```

Comment:
i would need some 👁️  for this copy. Intent is to inform the user that if using the flag but no configmap is found, then they should not use the flag and make a regular `okteto pipeline deploy`

cc @omnomagonz for some copy 🪄 magic